File: Bekend/CarServiceApi/Models/Car.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CarServiceApi.Models
{
    public class Car
    {
        [Key]

        public int Id { get; set; }
        public string Location { get; set; }
        public int PricePerDay { get; set; }
        public int NumberOfSeats { get; set; }
        public int BabySeats { get; set; }
        public string  Brand { get; set; }
        public int Year { get; set; }
        public string Model { get; set; }

        public ICollection<CarReservation> CarReservations { get; set; }

        public ICollection<CarMark> Marks { get; set; }

        
        public int RentCarCompanyId { get; set; }


    }
}

File: projekat/src/app/entities/car/car.ts
export class Car{
    id: number;
    location: string;
   
    brand: string;
    model: string;
    year: number;
    pricePerDay:number;
    numberOfSeats: number;
    mark=0;
    
    
    babySeats: number;
   
    currentlyAvailable: boolean;
    
    

    constructor(id: number,location: string,brand: string,model: string,year: number,pricePerDay: number,available: boolean,babyseats:number,numberOfSeats: number)
    {
        this.id = id;
        this.location = location
        
        this.brand = brand;
        this.model = model;
        this.year = year;
        this.pricePerDay = pricePerDay;
        this.currentlyAvailable = available;
        this.babySeats = babyseats;
       this.numberOfSeats = numberOfSeats;
       
    }
}